Libraries/Lists/VirtualizedList.js

```diff
@@ -1013,10 +1013,7 @@ class VirtualizedList extends React.PureComponent<Props, State> {
 
   _defaultRenderScrollComponent = props => {
     const onRefresh = props.onRefresh;
-    if (this._isNestedWithSameOrientation()) {
-      // $FlowFixMe - Typing ReactNativeComponent revealed errors
-      return <View {...props} />;
```

Comment:
I think you just need to add a style with `flexDirection: 'row'` when horizontal is true.

Reply:
Thanks for the feedback @sahrens! I'll give that a go and report bak if it works.